### DISCUSSION TITLE
close/retire all but one of the forums

### DISCUSSION THREAD

**Discussion by VladimirAlexiev** [plantuml/plantuml#1371] [Ideas]
To emphasize what @The-Lum says in https://github.com/plantuml/plantuml/discussions/655 "where to request" and @evantill says on discord "which on between issues and discussions ?"

Currently we have 5 (not 4) places where people communicate:
- https://forum.plantuml.net/ ([mirrored on discord](https://discord.com/channels/1083727021328306236/1085148406114160710))
- https://github.com/plantuml/plantuml/issues
- https://github.com/plantuml/plantuml/discussions ([mirrored on discord](https://discord.com/channels/1083727021328306236/1084804443645685771))
- https://github.com/plantuml/backlog
- https://discord.gg/sXhzexAQGh

This is too much, and is harmful to the community since it leads to scattering of knowledge/effort.
@arnaudroques could you please:
- decide between "forum" and "discussions" and "discord". I vote for "discord"
- retire the other forums
- ideally, if possible, migrate the content from the retired forums to the selected one
- write clear guidelines when to post "issues" vs "discussions"
- put in the documentation and on all forums (retired and current) **prominent** links to the other forums, and to the guidelines. Eg here in "discussions" there's no link to discord (should be in "Announcements")

Needless to say, the retired forums should remain available read-only, because there are hundreds of links to them.

I haven't used discord much, but it's very hot and advanced, so I guess that is the best choice.
- pro: You can point to a particular post, eg [Jdriele: Hi everyone, I discover yesterday PlantUML by using XRM toolbox for Dataverse data modelling. This tool is crazy.](https://discord.com/channels/1083727021328306236/1083727025954627667/1091280298987225140)
- cons: it highlights the post for a very short time...

**Comment by VladimirAlexiev** [comment]
Note: I have added links to Discord channels that mirror some of the other fora.

**Comment by claudio-salvio** [comment]
@VladimirAlexiev 

Even recognizing the merits of Discord, if voting is an option, in principle I vote for:
[https://github.com/plantuml/plantuml/issues](https://github.com/plantuml/plantuml/issues)
[https://github.com/plantuml/plantuml/discussions](https://github.com/plantuml/plantuml/discussions)
[https://github.com/plantuml/backlog](https://github.com/plantuml/backlog)

I consider that each of them has a specific role, and the fact of focusing all the project activity on GitHub has more weight.

If it were possible to make modifications, I think it would be better to manage the backlog as a label in [issues](https://github.com/plantuml/plantuml/issues) and avoid using the [backlog](https://github.com/plantuml/backlog) repository.

Best regards,
Claudio Salvio

**Comment by jcralbino** [comment]
Altough I am user without using used discord, I believe GitHub provides all the needs for the project.

I agree they focusing the community in one platform would be beneficial 

I would vote to use GitHub discussion / issues only

**Comment by Evan0000000000** [comment]
The forum seems to have quite a lot of ads (and in my case from a work laptop, lots of firewall block warnings) so I'd be quite happy to see more discussion happen on the github. 

Using discord from work is problematic for a lot of people so it seems preferable to avoid that.

**Reply by arnaudroques** [reply]
> The forum seems to have quite a lot of ads

Authenticated user are ads free :-)
https://forum.plantuml.net/register
